src/style.css

```diff
@@ -1 +1,100 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 450;
+  
+}
+
+body {
+
+  margin: 0;
+
+}
+
+ul {
+
+  margin: 0;
+  
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.image {
+
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+
+  display: flex;
+  list-style-type: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav__item {
+  
+  display: flex;
+  align-items: center;
+  margin: 0 5px;
+  padding: 0 5px;
+  position: relative;
+  
+}
+
+.nav__item:first-child {
+
+  padding-left: 0;
+  margin-left: 0;
+
+}
+
+.is-active::after {
+
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 50%;
+  transform: translateX(-51%);
+  height: 4px;
+  width: 100%;
+  background-color: #00acdc;
+  border-radius: 8px;
+  
+}
+
+.nav__item:last-child {
+
+  margin-right: 0;
+  padding-right: 0;
+}
```

Comment:
when you set margins to space elements, only set it on one side, this way you won't need to reset it for the first and last children. only for one